helm/fcp-fd-data/values.yaml

```diff
@@ -58,3 +58,8 @@ usePodIdentity: true
 azureIdentity:
   resourceID: not-a-real-clientID
   clientID: not-a-real-clientID
+
+ingress:
+  class: nginx
+  endpoint: ffc-fd-data
```

Comment:
Endpoint should be `fcp-fd-data` to match service name?

Reply:
good spot!